### ISSUE TITLE
Tutorial for training loop w/o assistant

### ISSUE THREAD

**Issue by mathisrichter** [lava-nc/lava-dl#154]
### User story

As a user, I want to have a tutorial that explains how to implement a training loop without the assistant to **\<missing benefit\>**.

**Comment by mathisrichter** [comment]
@bamsumit Could you check what the benefit would be to be able to not use the assistant? Does this unlock more features or make users more flexible in implementing the training loop?

Also, could you add a priority label? It is probably not urgent but if this is core feature, I would say clear documentation is important. If the feature is more exotic, I would think it's not so important.

**Comment by bamsumit** [comment]
@mathisrichter The [MNIST training tutorial](https://github.com/lava-nc/lava-dl/blob/main/tutorials/lava/lib/dl/bootstrap/mnist/train.ipynb) already shows how the training can be done without the `assistant`. It is the same steps as training a standard PyTorch network, so it would not add much. Not using the `assistant` would perhaps clarify how PyTorch training is done, nothing specific to lava-dl. I think it will do a good job of educating people and make it easier to tweak some aspects of the training. I clearly remember many people found having the loops in the example jarring (I personally don't think so) when we reviewed the tutorials, which actually lead to the work on assistants.

I would keep this as a very low priority for now.
